### ISSUE TITLE
[$10][MSFT-221] User only can focus on "Email address" text-box and can't focus on other items using tab

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2880]
**Steps to Reproduce**
1. Go to: https://accounts.topcoder.com/member/forgot-password
2. Use the tab key to focus on individual items in page

**Expected Result**
User should be able to select all items using tab

**Actual Result**
User only can focus on "Email address" text-box and can't focus on other items using tab ( ex: "GET RESET LINK" button, "Back to Login" link,...)

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F26

**Comment by romitgit** [comment]
PR for this issue : https://github.com/appirio-tech/accounts-app/pull/238

**Comment by lakshmiathreya** [comment]
Verified on Win 10 Edge, Firefox and Chrome/MacOS Chrome.

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096760 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096760 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
@romitgit, please sign-up with Topcoder X tool<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096760 has been updated - it has been assigned to Timor.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by tosha5252** [comment]
In production env, tab still skips 'get reset link' and 'back to login'

**Comment by nkumar-topcoder** [comment]
@tosha5252 can you pls try browser incognito mode for chrome and edge please.

**Comment by crazyk07** [comment]
@nkumar-topcoder  Partially confirmed what @tosha5252  said:

What works:
1. Tab button does go through each selection (Works on a screenreader as well)

What does NOT Work:
1. "Back to Login" link is not **Visually** highlighted when it is actually highlighted.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096760 has been updated - it has been assigned to sexta13.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by sexta13** [comment]
@crazyk07 I've created a PR to fix what's not working

**Comment by crazyk07** [comment]
#283

**Comment by nkumar-topcoder** [comment]
@lakshmiathreya @drasticdpk merged to `qa-accessibility` branch. In case if you have issue, let me know then I will merge to `dev` branch

**Comment by drasticdpk** [comment]
Verified on Chrome browser.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096760<br/><br/>```This is an automated message for crazyk via Topcoder X```
